pkg/services/team/teamimpl/store.go

```diff
@@ -336,6 +337,27 @@ func (ss *xormStore) GetByUser(ctx context.Context, query *team.GetTeamsByUserQu
 	return queryResult, nil
 }
 
+// GetIDsByUser is used by the Guardian when checking a users' permissions
+func (ss *xormStore) GetIDsByUser(ctx context.Context, query *team.GetTeamIDsByUserIDQuery) ([]int64, error) {
+	queryResult := make([]int64, 0)
+	err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		var params []any
+		params = append(params, query.UserID, query.OrgID, query.OrgID)
+		return sess.SQL(`SELECT t.id
```

Comment:
Do we even need the join to the team table? We have the team ID using `tm.team_id`

Reply:
That's a good shout! Could just filter by user ID and org ID from the team members table.

Reply:
Well spotted, thanks for taking a look :)